README.md

```
@@ -53,31 +53,28 @@ NocoDB is the fastest and easiest way to build databases online.
-  nocodb/nocodb:latest
+## Docker
+
+A single image that contains 
+- 🇳  NocoDB,
+- 🐘 PostgreSQL,
+- ⚡ Valkey(Redis), 
+- 🌐 nginx gateway.
+- 🔒 automated SSL certs and auto renew
+read more on our [wiki](/getting-started/self-hosted/installation/docker-aio)
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)